src/snowflake/snowpark/functions.py

```diff
@@ -7208,7 +7208,7 @@ def udf(
     """
 
     # Initial check to make sure no unexpected args are passed in
-    check_decorator_args(kwargs)
+    check_decorator_args(**kwargs)
```

Comment:
Nit: unboxing kwargs in `check_decorator_args `  isn't necessary. I'm OK if you don't change it becauswe we'll change this part pretty soon.